client/app.jsx

```diff
@@ -54,7 +54,10 @@ export default class App extends React.Component {
   }
 
   render() {
-    const title = this.state.route.path.split('/').splice(1, 2).join(' ').toUpperCase();
+    let title = this.state.route.path.split('/').splice(1, 2).join(' ').toUpperCase();
+    if (title === '') {
+      title = 'FITNESS REPORT';
+    }
```

Comment:
It would probably make more sense if each page just had a header in it with a hard-coded title.